pkg/kando/location_pull.go

```diff
@@ -73,7 +73,7 @@ func runLocationPull(cmd *cobra.Command, args []string) error {
 		if err = connectToKopiaServer(ctx, p); err != nil {
 			return err
 		}
-		return kopiaLocationPull(ctx, kopiaSnap.ID, s, target)
+		return kopiaLocationPull(ctx, kopiaSnap.ID, s, target, p.Credential.KopiaServerSecret.Password)
```

Comment:
```suggestion
		return kopiaLocationPull(ctx, kopiaSnap.ID, s, p.Credential.KopiaServerSecret.Password, target)
```
Just regrouping the string args